apollo-gradle-plugin/testProjects/README.md

```diff
@@ -0,0 +1,13 @@
+# testProjects
+
+This directory contains projects used by the plugin tests.
+
+You can find other files in [../src/test/files](../src/test/files). These projects are programmatically constructed but it becomes harder and harder to maintain so new test projects should be added here. It will duplicate some files and schemas but git should do the correct thing and only store objects once.
```

Comment:
@tasomaniac if ok with you, I'll switch to using plain test directories for plugin tests instead of programmatically constructing them like it was the case before. It's just easier to maintain and debug.

Reply:
Let's try and see. In other projects, I've also made them programmatically since they are dynamic and some pieces needs to change.

For the first time here, I've seen the approach is slightly different. I think it makes sense.